service-front/module/Application/view/application/pages/how_will_the_donor_confirm.twig

```diff
@@ -0,0 +1,55 @@
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <form method="POST">
+            <h1 class="govuk-heading-xl">How will the donor confirm their identity</h1>
+
+            <h2 class="govuk-heading-m">ID over the phone (preferred)</h2>
+            <div class="govuk-form-group">
+                <div class="govuk-radios" data-module="govuk-radios">
+                    {% for datum in options_data %}
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" id="{{ datum }}" name="id_method" type="radio"
+                                   value="{{ datum }}">
+                            <label class="govuk-label govuk-radios__label" for="{{ datum }}">{{ datum }}</label>
+                        </div>
+                    {% endfor %}
+                </div>
+            </div>
+            <div class="govuk-form-group">
+                <h2 class="govuk-heading-m">ID in person at the Post Office (if unable to ID over the phone)</h2>
+
+
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="whereDoYouLive" name="id_method" type="radio"
+                           value="england">
+                    <label class="govuk-label govuk-radios__label" for="whereDoYouLive">
```

Comment:
Appreciate these aren't used yet, but they all have the same ID and the values don't make any sense. Would be good to make this more representative.